core/integration/features/rewards/realised_returns.feature

```diff
@@ -0,0 +1,368 @@
+      | aux2                                                             | USDT.0.1 | 1000000000 |
+      | party1                                                           | USDT.0.1 | 1000000000 |
+      | party2                                                           | USDT.0.1 | 1000000000 |
+      | party3                                                           | USDT.0.1 | 1000000000 |
+      | party4                                                           | USDT.0.1 | 1000000000 |
+      | a3c024b4e23230c89884a54a813b1ecb4cb0f827a38641c66eeca466da6b2ddf | USDT.0.1 | 1000000000 |
+    And the markets:
+      | id       | quote name | asset    | risk model                    | margin calculator         | auction duration | fees         | price monitoring | data source config     | linear slippage factor | quadratic slippage factor | sla params      | decimal places | position decimal places |
+      | ETH/USDT | USDT.0.1   | USDT.0.1 | default-log-normal-risk-model | default-margin-calculator | 1                | default-none | default-none     | default-eth-for-future | 1e-3                   | 0                         | default-futures | 0              | 0                       |
+
+    # Exit opening auction and close the positions of the auxiliary parties so they don't receive rewards
+    Given the parties place the following orders:
+      | party | market id | side | volume | price | resulting trades | type       | tif     |
+      | aux1  | ETH/USDT  | buy  | 1000   | 1     | 0                | TYPE_LIMIT | TIF_GTC |
+      | aux1  | ETH/USDT  | buy  | 1      | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+      | aux2  | ETH/USDT  | sell | 1      | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+      | aux2  | ETH/USDT  | sell | 1000   | 1999  | 0                | TYPE_LIMIT | TIF_GTC |
+    When the opening auction period ends for market "ETH/USDT"
+    And the parties should have the following profit and loss:
+      | party | volume | unrealised pnl | realised pnl |
+      | aux1  | 1      | 0              | 0            |
+      | aux2  | -1     | 0              | 0            |
+    And the trading mode should be "TRADING_MODE_CONTINUOUS" for the market "ETH/USDT"
+    Given the parties place the following orders:
+      | party | market id | side | volume | price | resulting trades | type       | tif     |
+      | aux1  | ETH/USDT  | sell | 1      | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+      | aux2  | ETH/USDT  | buy  | 1      | 1000  | 1                | TYPE_LIMIT | TIF_GTC |
+    When the network moves ahead "1" blocks
+    And the parties should have the following profit and loss:
+      | party | volume | unrealised pnl | realised pnl |
+      | aux1  | 0      | 0              | 0            |
+      | aux2  | 0      | 0              | 0            |
+
+
+#   Scenario: In a falling market where short positions are profitable, parties have only unrealised pnl (0056-REWA-118)(0056-REWA-129)
```

Comment:
Have checked locally and they are all passing for me but these should be uncommented now.